src/ai/ai_config.cpp

```diff
@@ -118,3 +118,13 @@ void AIConfig::SetSetting(const char *name, int value)
 
 	ScriptConfig::SetSetting(name, value);
 }
+
+void AIConfig::AddRandomDeviation()
+{
+	int start_date = this->GetSetting("start_date");
+
+	ScriptConfig::AddRandomDeviation();
+
+	/* start_date = 0 is a special case, where random deviation does not occur. */
+	if (start_date == 0) this->SetSetting("start_date", start_date);
```

Comment:
this->SetSetting("start_date", 0)

Reply:
Maybe just something like
```suggestion
	if (start_date != 0) return;
	this->SetSetting("start_date", 0);
```